src/DataGrid.tsx

```diff
@@ -228,15 +223,10 @@ function DataGrid<R, SR>({
   const [scrollLeft, setScrollLeft] = useState(0);
   const [columnWidths, setColumnWidths] = useState<ReadonlyMap<string, number>>(() => new Map());
   const [selectedPosition, setSelectedPosition] = useState<SelectCellState | EditCellState<R>>({ idx: -1, rowIdx: -1, mode: 'SELECT' });
-  const [copiedPosition, setCopiedPosition] = useState<Position & { value: unknown } | null>(null);
+  const [copiedCell, setCopiedCell] = useState<{ row: R; column: CalculatedColumn<R, SR> } | null>(null);
```

Comment:
I think we only need to save the column key.
```suggestion
  const [copiedCell, setCopiedCell] = useState<{ row: R; columnKey: string } | null>(null);
```

Reply:
Need `idx` to set styles